src/monitor/monitor.c

```diff
@@ -231,10 +231,10 @@ static int client_registration(struct sbus_request *dbus_req, void *data)
     DBusError dbus_error;
     dbus_uint16_t svc_ver;
     dbus_uint16_t svc_type;
+    dbus_uint16_t explicitly_configured = true;
     char *svc_name;
     dbus_bool_t dbret;
     int ret;
-    bool explicitly_configured = true;
 
```

Comment:
Ouch. You're right Pavel. The patch I posted works as expected and I'm updating the series and it's the only change for now.